package.json

```diff
@@ -11,7 +25,8 @@
     "build": "npm run clean && npm run build-babel",
     "devstart": "nodemon --exec babel-node src/index.js",
     "lint": "eslint --fix ./src/**/*.js",
-    "test": "nyc mocha --require @babel/register --require @babel/polyfill --timeout 1000 --exit"
+    "test": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text mocha --require @babel/register --require @babel/polyfill --exit",
```

Comment:
- What's your reason for having `cross-env`?
- I don't think you should have this `--reporter=lcov --reporter=text` in your `test script`. Only coverage should have it

Reply:
`nyc` collect all the coverage report. 
`babel-plugin-istanbul` respects the include/exclude configuration options from `nyc`, but you also need to configure NYC not to instrument your code by adding these settings in your package.json:
see [Babel-plugin-instanbul](https://github.com/istanbuljs/babel-plugin-istanbul) for more details

Reply:
Done. I have removed --reporter=lcov --reporter=text.